wazo_acceptance/steps/blocklist.py

```diff
@@ -0,0 +1,62 @@
+from behave import given, step, then
+from hamcrest import assert_that, contains_inanyorder, has_entries
+
+
+# e.g. When the user "Charlie Unknown" adds number "+15555555555" with label "bad" to their blocklist
```

Comment:
Is this comment really necessary? We can grep in the features to get an usage example.